model/infrastructure/Catalog.ttl

```diff
@@ -33,27 +33,65 @@ ids:ResourceCatalog
 .
 
 ids:ParticipantCatalog
-    rdfs:subClassOf ids:Catalog;
     a owl:Class;
+    rdfs:subClassOf ids:Catalog;
     rdfs:label "Participant Catalog"@en ;
-    rdfs:comment "Class that aggregates Participants which, as a whole, from a (distributable) Catalog."@en ;
+    rdfs:comment "Class that aggregates Participants, as a whole, from a (distributable) Catalog."@en ;
     idsm:validation [
         idsm:forProperty ids:member;
         idsm:relationType idsm:OneToMany;
     ];
 .
 
 ids:ConnectorCatalog
-    rdfs:subClassOf ids:Catalog;
     a owl:Class;
+    rdfs:subClassOf ids:Catalog;
     rdfs:label "Connector Catalog"@en ;
-    rdfs:comment "Class that aggregates Connectors which, as a whole, from a (distributable) Catalog."@en ;
+    rdfs:comment "Class that aggregates Connectors, as a whole, from a (distributable) Catalog."@en ;
```

Comment:
Linguistic comment: I would remove those commas completely.

Reply:
Also, other places in this file

Reply:
Done. See 2164269